resources/nl-core/nl-core-HealthProfessional-PractitionerRole.xml

```diff
@@ -18,6 +18,12 @@
   <purpose value="A derived profile from [zib-HealthProfessional-PractitionerRole](http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-PractitionerRole) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
   <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
   <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="ART-DECOR-hg" />
+    <uri value="https://decor.nictiz.nl/ad/#/hg-/datasets/dataset/2.16.840.1.113883.2.4.3.11.60.103.1.1/2020-10-19T17:52:39" />
+    <name value="Huisarts- en paramedische overdrachten" />
+    <comment value="This is a preliminary mapping that should not be used for immediate implementation." />

```

Comment:
De comment is misschien onwenselijk om in nl-core te plaatsen. Mogelijk moeten we de nl-core-mappings pas doen als we er meer zeker van zijn?

Reply:
De eisen aan de naamgeving zijn opgenomen in de profiling guidelines: https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#Associating_the_functional_definition_to_StructureDefinitions